### ISSUE TITLE
Outline configuration 

### ISSUE THREAD

**Issue by boostedcitizens** [sailro/EscapeFromTarkov-Trainer#13]
Hi! Sorry to bother again but is there a way to change the opacity outline of the wallhack?

**Comment by sailro** [comment]
Good question,

I think given we use a RGBA for colors, it can perhaps work out of the box, just use an alpha with a lower value:

like in the default trainer.ini:

```
EFT.Trainer.Features.Players.BearColor=[0.0,0.0,1.0,1.0]
EFT.Trainer.Features.Players.BossColor=[1.0,0.0,0.0,1.0]
EFT.Trainer.Features.Players.Enabled=true
EFT.Trainer.Features.Players.Key="F5"
EFT.Trainer.Features.Players.ScavColor=[1.0,0.921568632,0.0156862754,1.0]
EFT.Trainer.Features.Players.UsecColor=[0.0,1.0,0.0,1.0]
```

See BossColor (full red by default):
`BossColor=[1.0,0.0,0.0,1.0]`
Red: 1.0
Green: 0.0
Blue: 0.0
Alpha: 1.0

So try 
`BossColor=[1.0,0.0,0.0,0.5]` to see if it works ?

**Comment by boostedcitizens** [comment]
Hi Sailro,
I tried changing the Alpha value to 0.5. It somewhat works but not as I thought it would. So I played around with different values like 0.1 which just makes all the bots turn see through red. I increased the value to 0.3 and it mostly turns all the bots to yellow except for the bears. but 0.5 works well. I recommend changing the value to about 0.6 or 0.7.  

Once again thank you for your help! Much love! :)
